.github/workflows/ci.yml

```diff
@@ -32,6 +34,8 @@ jobs:
     - uses: actions/checkout@v3
     - name: Install Nix
       uses: DeterminateSystems/nix-installer-action@main
+    - name: Install magic Nix cache
+      uses: DeterminateSystems/magic-nix-cache-action@main
       with:
```

Comment:
This `with` was moved incorrectly from the nix installation to the magic nix cache installation, that's why it's failing.